docker-multinode/common.sh

```diff
@@ -47,10 +51,7 @@ kube::multinode::main(){
   CURRENT_PLATFORM=$(kube::helpers::host_platform)
   ARCH=${ARCH:-${CURRENT_PLATFORM##*/}}
 
-  DEFAULT_NET_INTERFACE=$(ip -o -4 route show to default | awk '{print $5}' | head -1)
-  NET_INTERFACE=${NET_INTERFACE:-${DEFAULT_NET_INTERFACE}}
-
-  DEFAULT_IP_ADDRESS=$(ip -o -4 addr list ${NET_INTERFACE} | awk '{print $4}' | cut -d/ -f1 | head -1)
+  DEFAULT_IP_ADDRESS=$(ip -o -4 addr list $(ip -o -4 route show to default | awk '{print $5}' | head -1) | awk '{print $4}' | cut -d/ -f1 | head -1)
```

Comment:
I've tested your changes with Vagrant. There is one problem. Vagrant VMs have the same IP address for the first interface. This causes problem when IP is passed to  this parameter `--hostname-override=${IP_ADDRESS} ` Finally the node is unreachable. You should add possibility to chose interface for `DEFAULT_IP_ADDRESS`. For instance you can check if variable `NET_INTERFACE` is set and take a proper action